src/main/java/io/servertap/api/v1/ServerApi.java

```diff
@@ -206,6 +204,111 @@ public static void saveWorld(Context ctx) {
         ctx.json("success");
     }
 
+    private static void addFolderToZip(File folder, ZipOutputStream zip, String baseName, String rootName) throws IOException {
+        File[] files = folder.listFiles();
+        assert files != null;
+        for (File file : files) {
+            if (file.isDirectory()) {
+                if(rootName == null) {
+                    addFolderToZip(file, zip, baseName, folder.getName());
+                } else {
+                    addFolderToZip(file, zip, baseName, rootName);
+                }
+            } else {
+                String name = file.getAbsolutePath().substring(baseName.length())
+                        // Trim first slash (absolute path)
+                        .replaceFirst("^/", "");
+//              Join path and folder name
+                if(rootName == null) {
+                    name = folder.getName() + "/" + name;
```

Comment:
Everywhere you use a literal `/`, you should change it to `File.pathSeparator` to make sure it will work on Windows/macOS/Linux

Reply:
derp: `File.separator`, my bad.